react-notes/src/components/overlayingAlert/overlayingAlert.jsx

```diff
@@ -0,0 +1,25 @@
+import Alert from '@material-ui/lab/Alert';
+import { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+
+const OverlayingAlert = ({ children, status }) => {
+  const [open, setOpen] = useState(false);
+  useEffect(() => {
+    setOpen(true);
+  }, []);
+
+  return (
+    open && (
+      <Alert severity={status} onClose={() => setOpen(false)}>

```

Comment:
```suggestion
      <Alert severity={status} onClose={setOpen}>
```
And use please prev state inside the `setOpen` function